test/parse-result.js

```diff
@@ -210,23 +254,6 @@ describe('Parse result namespace', () => {
     it('should have a convenience method for retrieving source map', () => {
       expect(element.sourceMapValue).to.deep.equal([[1, 2]]);
     });
-
-    it('should serialize with a sourceMap attribute', () => {
```

Comment:
This test is duplicate and it's effectively the exact same thing as `should round-trip correctly` test above.